packages/babel-core/package.json

```diff
@@ -53,6 +53,49 @@
     "@babel/helper-module-transforms": "workspace:^",
     "@babel/helpers": "workspace:^",
     "@babel/parser": "workspace:^",
+    "@babel/plugin-proposal-async-generator-functions": "workspace:^",
```

Comment:
Q: Should we also internalize `@babel/plugin-bugfix-safari-id-destructuring-collision-in-function-expression` and `@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining`?

Since we will defaults to `bugfixes: true` in Babel 8, I think they should be included in internal plugin set.

Reply:
Done :+1: I've left `@babel/preset-modules` out for now; we can decide what to do with it in the future :)